requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ pysrt==0.4.7
 PyYAML==3.10
 requests==2.7.0
 requests-oauthlib==0.4.1
+rpdb==0.1.6
```

Comment:
@mollydb This new tool I'm considering is ISC licensed, any concerns there?

https://pypi.python.org/pypi/rpdb/